src/minisatip.c

```diff
@@ -1908,7 +1920,11 @@ void http_response(sockets *s, int rc, char *ah, char *desc, int cseq, int lr)
 		(lresp == sizeof(resp) - 1) ? "(message truncated) " : "", s->sock,
 		get_sockaddr_host(s->sa, ra, sizeof(ra)), get_sockaddr_port(s->sa),
 		lr, s->id);
-	LOGM("MSG client << process :\n%s", resp);
+	//LOGM("MSG client << process :\n%s", resp);
```

Comment:
remove this

Reply:
You have multiple log lines in read_rtsp, do you mind updating all of them ?

Reply:
> remove this

This is not an error. I leave here to understand the next block. However, if you want I'll remove it.

Reply:
> You have multiple log lines in read_rtsp, do you mind updating all of them ?

You mean change from LOG to LOGM in the `read_rtsp()` function?